gitdash/README.md

```diff
@@ -0,0 +1,58 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First install the dependencies for the project:
+
+```bash
+cd gitdash
+npm install
+```
+
+Then, run the development server:
+
+```bash
+npm run dev
+# or
+yarn dev
+```
+
+To get the GitHub authentication to work, you need to create a `.env` file in the `gitdash` directory with the following content
+
+```
+GITHUB_ID=<your_github_client_id>
+GITHUB_SECRET=<your_github_client_secret>
+```
+
+Follow the guide [here](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app) to create a new GitHub OAuth app.
+
+The details should look something like this:
+
+![OAuth App Details](https://i.imgur.com/p2yQzuT.png)
+
+On creating the OAuth app on GitHub, you need to set the `GITHUB_ID` in the `.env.local` file as the Client ID you see in the GitHub OAuth App in your account, and the `GITHUB_SECRET` to the Client Secret that you generate. Refer to the image below for more details
+
+![GitHub OAuth App Creds](https://i.imgur.com/NurdT2w.png)
```

Comment:
Think about including these images in the repository rather than hosting them on Imgur. If Imgur ever goes down, then so, too, will your README!

Reply:
Missed this comment. Will include it in the final changes.